recipes/lsd/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "lsds" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/funkelab/lsd/archive/v{{ version }}.tar.gz
+  sha256: b1afe5e78e1ef21fbc4b03b86de3577b74bb0533f056d75d339d9b6ce0f1c2ed
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - cython
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - h5py
+    - scikit-image
+    - requests
+    - cython
+    - gunpowder
+test:
+  imports:
+    - lsd
+  commands:
+    - pip check
+  requires:
+    - pip
```

```suggestion
  source_files:
    - tests
  commands:
    - pip check
    - pytest tests
  requires:
    - pip
    - pytest
```

See https://conda-forge.org/docs/maintainer/adding_pkgs.html#running-unit-tests

---

Unfortunately the tests are not ready to be included in the CI/CD. They rely on other packages that are not yet on pypi/conda-forge